pkgs/development/libraries/folks/default.nix

```diff
@@ -33,19 +32,14 @@ in stdenv.mkDerivation rec {
 
   passthru = {
     updateScript = gnome3.updateScript {
-      packageName = "folks";
-      attrPath = "gnome3.folks";
-      versionPolicy = "none";
+      packageName = pname;
```

Comment:
We should keep the version policy (we are using what seems to be unstable versions according to GNOME versioning).